lib/common/common/src/mmap_hashmap.rs

```diff
@@ -186,31 +193,20 @@ impl<K: Key + ?Sized> MmapHashMap<K> {
         self.header.buckets_count as usize
     }
 
+    pub fn keys(&self) -> impl Iterator<Item = &K> {
+        (0..self.keys_count()).filter_map(|i| {
+            let entry = self.get_entry(i).ok()?;
```

Comment:
Maybe we can squeeze a bit more performance if we'll seek over entries contents only rather than looking at the buckets table.

But we can do it later as a separate PR as further optimization.